File: scrudbeans-integration-tests-java/src/test/java/myjavapackage/test/SimpleErrorResponse.java
package myjavapackage.test;



import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;


@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class SimpleErrorResponse {

    private String message;

    private String remoteAddress;

    private String requestMethod;

    private String requestUrl;

    private Integer httpStatusCode;

    private String httpStatusMessage;

    private String userAgent;

    private Throwable throwable;


}

File: scrudbeans-integration-tests-kotlin/src/test/kotlin/mykotlinpackage/test/SimpleErrorResponse.kt
package mykotlinpackage.test

// import com.github.manosbatsis.scrudbeans.api.error.ConstraintViolationEntry

data class SimpleErrorResponse(
    var message: String? = null,
    var remoteAddress: String? = null,
    var requestMethod: String? = null,
    var requestUrl: String? = null,
    var httpStatusCode: Int? = null,
    var httpStatusMessage: String? = null,
    var userAgent: String? = null,
    var throwable: Throwable? = null,
    // var validationErrors: Set<ConstraintViolationEntry>? = null
)
